docs/source/projects.rst

```diff
@@ -349,103 +349,170 @@ useful if you quickly want to test a project in your existing shell environment.
-A brief overview of how to configure and use this feature is as follows:
-
-In project folder you need to create a ``backend_config.json`` with the following attributes:
-
-.. code-block:: json
-
-{
-  "kube-context": "docker-for-desktop",
-  
-  "image-uri": "username/mlflow-kubernetes-example",
-
-  "kube-job-template-path": "kubernetes_job_template.yaml"
-}
-
-The ``kube-context`` attribute is the kubernetes context where mlflow will run the Job. ``image-uri`` points to the 
-registry/repository/image where the image will be pushed so kubernetes can download it and run. Remeber that mlflow 
-expects that login credentials are already stored for both kubernetes context and docker repository to push images.
-
-The ``kube-job-template-path`` points to a yaml file with the kubernetes Job/Batch specification to run the traning on 
-kubernetes. 
-See below the example available in the Docker example project. For more information about specification options please see 
-Kubernetes docs:
-`Jobs - Run to Completion <https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/>`_
-
-Have in mind that mlflow overwrites the folowing attributes in the job yaml file so it can handle jobs creation and monitor its status:
+Run a Project on Kubernetes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+You can run MLflow Projects with :ref:`Docker environments <project-docker-container-environments>`
+on Kubernetes. The following sections provide an overview of the feature, including a simple
+Project execution guide with examples.
+
+To see this feature in action, you can also refer to the
+`Docker example <https://github.com/mlflow/mlflow/tree/master/examples/docker>`_, which includes
+the required Kubernetes backend configuration (``kubernetes_backend.json``) and `Kubernetes Job Spec
+<https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#writing-a-job-spec>`_
+(``kubernetes_job_template.yaml``) files.
+
+How it works
+~~~~~~~~~~~~
+
+When you run an MLflow Project on Kubernetes, MLflow constructs a new Docker image
+containing the Project's contents; this image inherits from the Project's
+:ref:`Docker environment <project-docker-container-environments>`. MLflow then pushes the new
+Project image to your specified Docker registry and starts a
+`Kubernetes Job <https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/>`_
+on your specified Kubernetes cluster. This Kubernetes Job downloads the Project image and starts
+a corresponding Docker container. Finally, the container invokes your Project's
+:ref:`entry point <running-projects>`, logging parameters, tags, metrics, and artifacts to your
+:ref:`MLflow tracking server <tracking_server>`.
+
+Execution guide
+~~~~~~~~~~~~~~~
+
+You can run your MLflow Project on Kubernetes by following these steps:
+
+1. First, add a Docker environment to your MLflow Project, if one does not already exist. For
```

Comment:
@dbczumar Could you remove the First, Then, and Finally from the steps 1, 2, and 4. They are really not needed since you are using an ordered list.

Reply:
Done! Thanks